types/sip.js/index.d.ts

```diff
@@ -321,13 +312,22 @@ declare namespace sipjs {
     }
 
     /* Header */
-    interface NameAddrHeader {
+    class NameAddrHeader {
         uri: string | URI;
         displayName: string;
 
+        constructor(uri: string | sipjs.URI, displayName: string, parameters: Array<{ key: string, value: string }>);
+        static parse(name_addr_header: string): sipjs.NameAddrHeader;
+
         setParam(key: string, value?: string): void;
         getParam(key: string): string;
         deleteParam(key: string): string;
         clearParams(): void;
     }
 }
+
+import SIP = sipjs;
+
+declare module 'sip.js' {
```

Comment:
Instead of using `declare module`, try this:
* Remove the `declare namespace sipjs` wrapper around everything.
* Add `export` keyword to everything.
* Add `export as namespace sipjs;` at the top to preserve the global `sipjs` variable.